pkgs/top-level/all-packages.nix

```diff
@@ -25161,6 +25161,8 @@ in
 
   protonmail-bridge = callPackage ../applications/networking/protonmail-bridge { };
 
+  protege = callPackage ../applications/science/logic/protege { };
```

Comment:
I am not sure if Protege should be lumped together with things like theorem provers, and it's more of a development/content creation tool than a scientific instrument. The base application doesn't even include a reasoner, for instance.

But this is just a matter of opinions.

Reply:
Agreed, putting it here was unintentional.